src/components/CreateGoals.js

```diff
@@ -15,23 +15,28 @@ function CreateApp() {
         Open Sauced is a tool to help track your open source contributions. You can get started by creating a goal
         workspace below.
       </p>
-      <p>A public repository name "open-sauced-goals" will be created on your GitHub account to store.</p>
+      <p>A public repository named "open-sauced-goals" will be created on your GitHub account to store data about your goals.</p>
       <small>
         <em>You own all your data saved while saucin.</em>
       </small>
     </React.Fragment>
   );
 }
 
-function InstallApp() {
+function InstallApp({user}) {
+  const repoUrl = `https://github.com/${user.login}/open-sauced-goals`;
```

Comment:
This should link to open-sauced/goals-template instead

Reply:
So we don't need the user context then?

Reply:
Wow, reviewing this on stream had me miss-read this. I event tested it wrong. This is good to go as is. The context of the user is needed.